pkg/constants/logMessages.go

```diff
@@ -19,91 +19,92 @@ package constants
 // constants for logger
 const (
 	//Debug Messages
+	DebugMessageCustomLoggerUsed                = "Custom logger used"
 	DebugMessageDevelopmentMode                 = "Development mode is : %v "
+	DebugMessageGettingStoredVariation          = "[%v] Got stored variation for User ID: %v of Campaign: %v as Variation: %v, found in UserStorageService"
+	DebugMessageGotVariationForUser             = "[%v] User ID: %v of Campaign: %v campaignType: %v got variation: %v inside method: %v "
+	DebugMessageImpressionForPush               = "[%v] impression built for pushing - AccountID: %v, UserID: %v, SID: %v, URL: %v, U: %v "
+	DebugMessageImpressionForTrackCustomGoal   	= "[%v] impression built for track goal -  AccountID: %v, UserID: %v, SID: %v, URL: %v, ExperimentID: %v, Combination: %v, GoalID: %v "
+	DebugMessageImpressionForTrackRevenueGoal  	= "[%v] impression built for track goal -  AccountID: %v, UserID: %v, SID: %v, URL: %v, ExperimentID: %v, Combination: %v, GoalID: %v, RevenueValue: %v "
```

Comment:
use` %+v` instead of `%v` for a better output, do not pass so many values for no reason.

Reply:
same applies for most of the log messages here